frontend/cypress/e2e/redirect.cy.js

```diff
@@ -33,11 +38,12 @@ describe("redirect", () => {
     cy.visit("/login");
     cy.url().should("include", "/posts");
     cy.logout();
+    cy.wait(5000);
   });
 
   it("redirects to login page if not logged in and tries to access posts route", () => {
     cy.visit("/posts");
-    cy.wait(1000);
+    cy.wait(5000);
```

Comment:
Can you explain this change please?

Reply:
I think cy.wait is a good practice. But with Cypress e2e test, sometimes it does not fully render ,then they proceed to execute cy.get(element), and that will lead to errors. So I added extra waiting time for the browser to fully loaded.